metricbeat/docs/running-on-kubernetes.asciidoc

```diff
@@ -88,16 +88,7 @@ spec:
 ===== Red Hat OpenShift configuration
 
 If you are using Red Hat OpenShift, you need to specify additional settings in
-the manifest file and enable the container to run as privileged.
```

Comment:
Are we removing this?

Reply:
I've checked it on Openshiftv4 (GCP setup) and with `privileged: true` and without `privileged: true` getting the same result:
from the container:
```
mount | grep /usr/share/metricbeat/data
/dev/sda4 on /usr/share/metricbeat/data type xfs (rw,relatime,seclabel,attr2,inode64,logbufs=8,logbsize=32k,prjquota)
```
no errors descibed here: https://github.com/elastic/beats/issues/17516#issue-595098235

additionally to that, I wanted to keep metricbeat and elastic-agent manifests (beside the actual configuration of modules/datasets) the same - https://github.com/elastic/observability-docs/pull/1498

Reply:
That's good! However maybe we need to be more explicit here and mention in which versions it is not required to avoid confusion?

Reply:
Should we rely on openshift version here instead of kubernetes version? according to this [doc](https://docs.openshift.com/container-platform/3.11/release_notes/ocp_3_11_release_notes.html#ocp-311-about-this-release) the last available openshift version 3.11 uses Kubernetes 1.11, which is not supported by us I think

Reply:
Strictly speaking yes we can claim that older versions are not supported however a small comment like `NOTE: for older versions of v4 users need to run the container as privileged` so as to provide a good hint to possible users that are still with older versions.

Reply:
I will add it.
I didn't check the metricbeat on openshift v3, but somehow I think it even might not work (at least some functionality) - issue described in this comment https://github.com/elastic/integrations/issues/2065#issuecomment-1002215969

Reply:
done - https://github.com/elastic/beats/pull/30054/commits/8d1e5d25e17794c52e854ad1c4b0eb7921d00860